### PR TITLE
tris game should be a state machine

### PR DESCRIPTION
**Pull request by sensorario**
- [x] Simone writes a new test in a Pull Request and sees that it fails.
- [x] Alessandro implements the code needed to pass the test.
- [ ] Simone approve pull request with 👍 
- [ ] Alessandro merge code on master

### CODE REVIEW

File: src/Tris.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+class Tris
+{
+	public function isIdle()
```
Comment (sensorario) [inline src/Tris.php]: As you can see [here](http://www.php-fig.org/psr/psr-2/#overview) "Code MUST use 4 spaces for indenting, not tabs."

Comment (sensorario) [inline src/Tris.php]: As you can see [here](http://www.php-fig.org/psr/psr-2/#overview) "Code MUST use 4 spaces for indenting, not tabs."

Comment (AlessandroMinoccheri) [inline src/Tris.php]: Fixed

**Comment by sensorario** [comment]
👍

**Pull request merged**